packages/@uppy/core/src/Uppy.ts

```diff
@@ -1724,10 +1724,10 @@ export class Uppy<M extends Meta, B extends Body = Record<string, never>> {
   /**
    * Find one Plugin by name.
    */
-  getPlugin(id: string): UnknownPlugin<M, B> | undefined {
+  getPlugin<T extends UnknownPlugin<M, B>>(id: string): T | undefined {
```

Comment:
Looks good, but it should default to `UnknownPlugin`

```suggestion
  getPlugin<T extends BasePlugin<M, B> = UnknownPlugin<M, B>>(id: string): T | undefined {
```

Reply:
I was actually thinking to not give it a default so it forces people to type the return type

Reply:
IMO that's going to frustrate users who are not interested in having strict types, we should provide a way to define strict types, but certainly not force it upon our users. Our users can decide for themselves if they want strict types or not.